client/blocks/importer/wordpress/import-everything/migration-error.tsx

```diff
@@ -0,0 +1,41 @@
+import { useChatWidget } from '@automattic/help-center/src/hooks';
+import { NextButton, SubTitle, Title } from '@automattic/onboarding';
+import { useTranslate } from 'i18n-calypso';
+import { useEffect } from 'react';
+
+interface Props {
+	siteUrl: string;
+	resetMigration: () => void;
+}
+export default function MigrationError( props: Props ) {
+	const translate = useTranslate();
+	const { openChatWidget } = useChatWidget();
+	const { siteUrl, resetMigration } = props;
+
+	useEffect( () => {
+		openChatWidget( {
```

Comment:
We can probably check if a user is eligible for a chat. Something like `useChatStatus` hook might be helpful, see the example: https://github.com/Automattic/wp-calypso/blob/trunk/packages/help-center/src/components/help-center-contact-form.tsx#L121-L130.

Also, if a user cannot connect to Zendesk, we might want to have a fallback and give them access to the contact form. Maybe a button that links to the form will help? Something from https://github.com/Automattic/wp-calypso/blob/trunk/client/components/chat-button/index.tsx#L112-L123 might help with this situation?